frontend/src/audio.html

```diff
@@ -0,0 +1,84 @@
+---
+pagination:
+  data: audios
+  size: 1
+  alias: audio
+permalink: "/audios/{{ audio.objectID | slug }}/"
+layout: base
+templateEngineOverride: md, njk, html
+eleventyComputed:
+  title: "{{ audio.objectID }}"
+---
+<section class="flex flex-col m-10">
+  <div class="flex flex-col">
+    <div class="flex align-baseline justify-between">
```

Comment:
Add comments before each block to help identifying them. See the `audio-item-template.js` for example.

Reply:
comments added